src/main/java/spark/RequestResponseFactory.java

```diff
@@ -26,6 +26,13 @@
     private RequestResponseFactory() {
     }
 
+    /**
+     * Solely used for custom error pages.
```

Comment:
It's for specifying that it only should be used from there. But when thinking about it this class should no nothing on which classes that uses it. I'll re-write it